### DISCUSSION TITLE
Continuous out of sync behaviour from filecoin chain

### DISCUSSION THREAD

**Discussion by hcm609** [filecoin-project/lotus#7627] [Build and Sync] [answered]
Hi.
Lately (after 12/Nov/2021) I am our of sync from the chain continuously.
The server is quite quick 64-Cores & Passmark 79000.
The rate to execute one epoch is around 1min/epoch or less.
It stops to (0 per second) for most of the time. 
The server has a load of 0.02 so it's not any other process that delays the whole process.

# lotus sync wait
Worker: 1119; Base: 1285353; Target: 1286077 (diff: 724)
State: message sync; Current Epoch: 1285462; Todo: 615
Validated 797 messages (0 per second)

Does anyone else experience the same out of sync issue.
Is there any fix on this?

**Comment by de-authority** [comment]
run `lotus net peers` , check if you have enough peers to sync.
and I think you may have bad hard disks

**Comment by hcm609** [comment]
lotus net peers|wc -l 
I have 157. So I need to change disks then.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
>The rate to execute one epoch is around 1min/epoch or less.

you need 30s or less to stay in sync. you get a new tipset every 30s - so you will fall behind with 1m or less

**Comment by LexLuthr** [comment, marked as answer]
@hcm609 Were you able to resolve this problem? If yes, then can you please let us know what was the cause of the delay?

**Reply by hcm609** [reply]
I am not sure what was the cause of that. By moving to a strong server and an NVME I see no complains
